### PR TITLE
update fancybox to 3.0

### PR DESCRIPTION
**Pull request by jjandxa**
screenshot_2017-06-01-21-26-51-918

### CODE REVIEW

**Comment by jjandxa** [comment]
@ivan-nginx why do it close?

**Comment by ivan-nginx** [comment]
@jjandxa because there are many conflicts and FB3.0 still in beta version.

File: _config.yml [hunk 1]
```diff
@@ -627,4 +627,4 @@ js: js
 images: images
 
 # Theme version
-version: 5.1.1
```
Comment (ivan-nginx) [inline _config.yml]: Why this diffs?

File: layout/_partials/head.swig [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
 
 {% if theme.fancybox %}
-  {% set fancybox_css_uri = url_for(theme.vendors._internal + '/fancybox/source/jquery.fancybox.css?v=2.1.5') %}
```
Comment (ivan-nginx) [inline layout/_partials/head.swig]: There is FB version in CSS?

File: layout/_scripts/vendors.swig [hunk 3]
```diff
@@ -14,7 +14,7 @@
 {% set js_vendors.velocity_ui = 'velocity/velocity.ui.min.js?v=1.2.1' %}
 
 {% if theme.fancybox %}
-  {% set js_vendors.fancybox  = 'fancybox/source/jquery.fancybox.pack.js?v=2.1.5' %}
```
Comment (ivan-nginx) [inline layout/_scripts/vendors.swig]: Version?

File: source/js/src/utils.js [hunk 4]
```diff
@@ -10,31 +10,25 @@ NexT.utils = NexT.$u = {
       .not('.group-picture img, .post-gallery img')
       .each(function () {
         var $image = $(this);
-        var imageTitle = $image.attr('title');
```
Comment (ivan-nginx) [inline source/js/src/utils.js]: There is `.find` function?

File: source/js/src/utils.js [hunk 5]
```diff
@@ -10,31 +10,25 @@ NexT.utils = NexT.$u = {
...
+        var imageTitle = $image.parents('article').find('.post-title-link').text();
+        if (!imageTitle) {
+          imageTitle = $image.parents('article').find('.post-title').text();
+        }
+        imageTitle.replace('/(^\s*)|(\s*$)/g', '');
         var $imageWrapLink = $image.parent('a');
 
         if ($imageWrapLink.size() < 1) {
-	        var imageLink = ($image.attr('data-original')) ? this.getAttribute('data-original') : this.getAttribute('src');
```
Comment (ivan-nginx) [inline source/js/src/utils.js]: There is my custom `data-original` function for lazyload?

File: source/lib/fancybox/.bower.json [hunk 6]
```diff
@@ -1,15 +0,0 @@
-{
```
Comment (ivan-nginx) [inline source/lib/fancybox/.bower.json]: Why delete file?

File: source/lib/fancybox/.gitattributes [hunk 7]
```diff
@@ -1,7 +0,0 @@
-# Auto detect text files and perform LF normalization
```
Comment (ivan-nginx) [inline source/lib/fancybox/.gitattributes]: Why delete file?

File: source/lib/fancybox/source/helpers/jquery.fancybox-buttons.css [hunk 8]
```diff
@@ -1,97 +0,0 @@
-#fancybox-buttons {
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/helpers/jquery.fancybox-buttons.css]: Why delete file?

File: source/lib/fancybox/source/helpers/jquery.fancybox-buttons.js [hunk 9]
```diff
@@ -1,122 +0,0 @@
- /*!
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/helpers/jquery.fancybox-buttons.js]: Why delete file?

File: source/lib/fancybox/source/helpers/jquery.fancybox-media.js [hunk 10]
```diff
@@ -1,199 +0,0 @@
-/*!
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/helpers/jquery.fancybox-media.js]: Why delete file?

File: source/lib/fancybox/source/helpers/jquery.fancybox-thumbs.css [hunk 11]
```diff
@@ -1,55 +0,0 @@
-#fancybox-thumbs {
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/helpers/jquery.fancybox-thumbs.css]: Why delete file?

File: source/lib/fancybox/source/helpers/jquery.fancybox-thumbs.js [hunk 12]
```diff
@@ -1,162 +0,0 @@
- /*!
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/helpers/jquery.fancybox-thumbs.js]: Why delete file?

File: source/lib/fancybox/source/jquery.fancybox.css [hunk 13]
```diff
@@ -1,274 +0,0 @@
-/*! fancyBox v2.1.5 fancyapps.com | fancyapps.com/fancybox/#license */
```
Comment (ivan-nginx) [inline source/lib/fancybox/source/jquery.fancybox.css]: Why delete file?

**Pull request closed without merging**